.github/workflows/on_pull_request.yml

```diff
@@ -291,7 +291,7 @@ jobs:
         with:
           path: |
             ~/.cache/ms-playwright
-          key: cache-playwright-linux-1.19.2
+          key: cache-playwright-linux-1.28.1
```

Comment:
Instead of hardcoding the Playwright version we could get it with an added step:
```
     - name: Get playwright version
        id: pw-version
        run: |
          echo "::set-output name=version::$(yarn playwright --version)"
        shell: bash

     - name: Cache playwright binaries
        uses: actions/cache@v2
        id: playwright-cache
        with:
          path: '~/.cache/ms-playwright'
          key: cache-playwright-${{ runner.os }}-${{ steps.pw-version.outputs.version }}
```

Reply:
thought: the risk with not hard-coding is it can upgrade to a version that breaks for us, which in my experience is more likely than out of date breakage. I could go either way though

Reply:
I think this is fine, mainly because it grabs the installed version number, and that is dictated by a pinned dependency in the package JSON file.